### PR TITLE
Make checking too many errors optionally

### PR DESCRIPTION
**Pull request by donggyu04**
1. Support ignore too many error as advanced configuration, instead of system configuration.

image

image

2. Add finished with warning status.

image

### CODE REVIEW

**Comment by donggyu04** [comment]
Support ignore too many error as advanced configuration, instead of system configuration.

image

File: ngrinder-controller/src/main/java/org/ngrinder/plugin/TooManyErrorCheckPlugin.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	}
+
+	@Override
+	public void sampling(ISingleConsole singleConsole,
+						 PerfTest perfTest,
+						 IPerfTestService perfTestService,
+						 ImmutableStatisticsSet intervalStatistics,
+						 ImmutableStatisticsSet cumulativeStatistics) {
+
+		if (!perfTest.getIgnoreTooManyError()) {
```
Comment (donggyu04) [inline ngrinder-controller/src/main/java/org/ngrinder/plugin/TooManyErrorCheckPlugin.java:36]: Check too many error in plugin.

File: ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestRunnable.java [hunk 2]
```diff
@@ -355,17 +355,16 @@ protected void addSamplingListeners(final PerfTest perfTest, final SingleConsole
 		singleConsole.addSamplingLifeCyleListener(new PerfTestSamplingCollectorListener(singleConsole,
 				perfTest.getId(), perfTestService, scheduledTaskService));
 		singleConsole.addSamplingLifeCyleListener(new AgentLostDetectionListener(singleConsole, perfTest,
-				perfTestService, scheduledTaskService));
+				perfTestService, scheduledTaskService));;
 		List<OnTestSamplingRunnable> testSamplingPlugins = pluginManager.getEnabledModulesByClass
-				(OnTestSamplingRunnable.class, new MonitorCollectorPlugin(config, scheduledTaskService,
-						perfTestService, perfTest.getId()));
+				(OnTestSamplingRunnable.class, asList(new MonitorCollectorPlugin(config, scheduledTaskService,
+					perfTestService, perfTest.getId()), new TooManyErrorCheckPlugin()));
```
Comment (donggyu04) [inline ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestRunnable.java:361]: Added `TooManyErrorCheckPlugin` as default plugin to sampling phase.

**Comment by songeunwoo** [comment]
good job^^

**Pull request merged**